src/go/configgenerator/route_generator.go

```diff
@@ -144,6 +147,63 @@ func MakeRouteConfig(serviceInfo *configinfo.ServiceInfo) (*routepb.RouteConfigu
 	}, nil
 }
 
+func MakePathRewriteConfig(method *configinfo.MethodInfo, httpRule *commonpb.Pattern) *prpb.PerRouteFilterConfig {
+	if method.BackendInfo == nil {
+		return nil
+	}
+
+	if method.BackendInfo.TranslationType == confpb.BackendRule_APPEND_PATH_TO_ADDRESS {
+		if method.BackendInfo.Path != "" {
+			return &prpb.PerRouteFilterConfig{
+				PathTranslationSpecifier: &prpb.PerRouteFilterConfig_PathPrefix{
+					PathPrefix: method.BackendInfo.Path,
+				},
+			}
+		}
+	}
+	if method.BackendInfo.TranslationType == confpb.BackendRule_CONSTANT_ADDRESS {
+		constPath := &prpb.ConstantPath{
+			Path: method.BackendInfo.Path,
+		}
+		if strings.ContainsRune(httpRule.UriTemplate, '{') {
```

Comment:
Add a unit test case of `CONSTANT_ADDRESS` with `constPath.UrlTemplate` is empty if it is possible?

Reply:
Added